### PR TITLE
Tidy up

### PR DESCRIPTION
**Pull request by NathanBaulch**
Just some minor tidy up - mostly spelling, grammar, newline inconsistencies, trailing whitespace, redundant types, etc.

### CODE REVIEW

File: retry_test.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func TestAttemptWithDelay(t *testing.T) {
 	})
 
 	is.Equal(iter1, 1)
-	is.Greater(dur1, 0*time.Millisecond)
+	is.GreaterOrEqual(dur1, 0*time.Millisecond)
```
Comment (ccoVeille) [inline retry_test.go:85]: How could be invalid?

I mean this test will always pass. A duration is always positive, no?

File: retry_test.go [hunk 2]
```diff
@@ -187,7 +187,7 @@ func TestAttemptWhileWithDelay(t *testing.T) {
...
```
Comment (ccoVeille) [inline retry_test.go:190]: Same?

File: retry_test.go (hunk 1 above)
Comment (NathanBaulch) [reply retry_test.go:85]: Could be my hardware or the fact that I'm running on Windows, but for me the first call to `AttemptWithDelay` (which succeeds immediately) returns within the minimum monotonic time period, so these tests always fail.

**Comment by samber** [comment]
Wow. That much??

Me, trying to write documentation in English before the Github Copilot era:

**Comment by samber** [comment]
(tests are broken)

**Comment by NathanBaulch** [comment]
Were you aware that I'm Australian before posting that gif? 😛 Don't worry, we don't take ourselves too seriously!
The unit tests should be fixed now.

**Comment by samber** [comment]
Ahah, sorry. I missed that little detail. 😳 🤣

**Pull request merged**